### PR TITLE
Implementing Clean Architecture

### PR DESCRIPTION
**Pull request by albertogalca**
Hi Tigran. As I promise, here my PR. I know this is overengineering, but I wanted to try and implement Clean Architecture (as people use to do in backend teams) into a frontend project. For next steps, I would like to add:

- Testing (snapshots and unit testing).
- Remove one `any` I have.
- Add more fields validations.

That's all. Hope you like it and don't hesitate to ask me whatever you need 😄.

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "crontab.am",
+  "name": "Cron expression generator",
```
Comment (tiggreen) [inline public/manifest.json:3]: Thanks!

File: src/ui/containers/Helper.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { connect } from "react-redux";
+
+import { Helper } from "../components";
+
+const mapStateToProps = () => ({
```
Comment (tiggreen) [inline src/ui/containers/Helper.tsx:5]: Very good, idea! 👍

**Review by tiggreen — APPROVED** [review]
Awesome PR! Thanks @albertogallego 🎉 🎉 I will merge it.

**Comment by tiggreen** [comment]
The only thing I'm trying to figure out @albertogallego is why [deploy preview](https://deploy-preview-1--objective-einstein-84b85f.netlify.com/) is failing to load. 🤔 Ideas?

**Comment by albertogalca** [comment]
@tiggreen I can see it 😄 . Just before merge it, try that all the behaviour is as you expected. I did some testing but maybe I'm missing something 👍 .

screen shot 2019-01-27 at 4 07 39 pm

**Pull request merged**